angular/components/team-efficiency/team-efficiency.js

```diff
@@ -80,3 +80,49 @@ angular.module('app.team', [])
             this.changeCurrentEmployee(employeeList[0]);
         }.bind(this));
     }]);
+
+    $(window).on("resize", function () {
+        if($("#team-sales").length !== 0) {
+            setTeamSalesChartLabelsStep();
+        }
+    });
+
+    $(function() {
+        if($("#team-sales").length !== 0) {
+            setTimeout(function() {
+                setTeamSalesChartLabelsStep();
+            }, 300)
+        }
+    });
+
+    function setTeamSalesChartLabelsStep() {
```

Comment:
This function can be refactored **$(window).width()** can be extracted alongside the **chart.setOptions** call